src/observable.js

```diff
@@ -114,6 +114,52 @@ extend(Observable.prototype, {
     return this._off(ANY, fn);
   },
 
+  subscribe(observer) {
+    const _this = this;
+    let closed = false;
+
+    if (typeof observer === 'function') {
+      observer = {
+        next: observer,
+        error: arguments[1],
+        complete: arguments[2]
+      };
+    }
+
+    if (observer.next) {
+      this.onValue(observer.next);
```

Comment:
> then we don't actually need to provide the toESObservable method.

Not exactly, `toESObservable` also adds `.takeErrors(1)` so the stream would end on the first error as spec requires. So we probably still need it as 100% compatible es-observable.

As for duplication, we probably could do something like `Observable.prototype.subscribe = ESObservable.prototype.subscribe` if they end up completely identical. Otherwise I'm ok with a bit of duplication.